dependencies/secret_service_test.go

```diff
@@ -0,0 +1,24 @@
+package dependencies_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/influxdata/flux/mock"
+)
+
+func TestSecret_Service(t *testing.T) {
+	ss := mock.SecretService{}
+	val, err := ss.LoadSecret(context.Background(), "key")
+	if err != nil {
+		t.Fatal(err)
+	}
+	if val != "val" {
+		t.Error("secret service returned wrong value")
+	}
+
+	_, err = ss.LoadSecret(context.Background(), "k")
```

Comment:
Nitpick: When you have something like this where the value doesn't have to be used outside of the if statement, it's preferred to do this:

```
if _, err := ss.LoadSecret(context.Background(), "k"); err == nil {
    ...
}
```